packages/bp/e2e/all.test.ts

```diff
@@ -46,19 +46,19 @@ const customTest = [test.auth, test.login, ...admin, ...studio, test.logout]
 
 const requestHandler = (req: HTTPRequest) => {
   if (shouldLogRequest(req.url())) {
-    console.info(`${getTime()} > REQUEST: ${req.method()} ${req.url()}`)
+    // console.info(`${getTime()} > REQUEST: ${req.method()} ${req.url()}`)
```

Comment:
Make sure to uncomment those lines as they seem to be used for debugging. I know it's not the best, maybe you have ideas on how we could improve this, if so please create some tasks with suggestions! 

If you want to disable the console output locally, simply run `jest` with `--silent`. E.g. `yarn test:e2e --silent`.